### ISSUE TITLE
Discord API v9

### ISSUE THREAD

**Issue by cwchristerw** [restcord/restcord#158]
I would like to have ability to use Discord threads and other newer features. I would like to do branch for API v9 compatibility :smile:

**Comment by cwchristerw** [comment]
How can I create service-description-v9.json? Do I need to make it manually?

**Comment by cwchristerw** [comment]
Using Swagger Spec Generator obviously
https://github.com/restcord/swagger-spec-generator

**Comment by cryptiklemur** [comment]
Yeah, that is the project thats used to generate a spec. Not sure if its currently working with the API. I haven't messed with it in a while.

**Comment by bumbummen99** [comment]
@cwchristerw Did you make any progress on this? I need this too as well as crosspost https://discord.com/developers/docs/resources/channel#crosspost-message

**Comment by LeigerGaming** [comment]
Looking to do crosspost also, based on this I suspect it's not possible with Restcord just yet :)

**Comment by bymem** [comment]
Any progress on this feature, it's sorely missed?

**Comment by pb-butcher** [comment]
Any progress on this feature? Otherwise i would start my own fork. I need events creating and handling

**Comment by cwchristerw** [comment]
There wasn't any progress I tried but it was too hard at that point when I started. I would suggest making new version compatible with Discord API v10 instead of Discord API v9.

**Comment by cwchristerw** [comment]
There is v9 branch in this repository probably it would be good idea to start there.

**Comment by cryptiklemur** [comment]
RestCord 0.9.0 supersedes the requested v9 work with Discord API v10 generated from the official OpenAPI specification. It includes thread, crosspost, and current channel operations. Migration guide: https://www.restcord.com/migration-0.9/
